tensorboard/plugins/profile_redirect/tf_profile_redirect_dashboard/tf-profile-redirect-dashboard.html

```diff
@@ -0,0 +1,110 @@
+<!--
+@license
+Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../paper-button/paper-button.html" />
+<link rel="import" href="../tf-imports/polymer.html" />
+
+<!--
+  A frontend that directs users to upgrade to the new version of the
+  profile plugin, now distributed separately from TensorBoard.
+-->
+<dom-module id="tf-profile-redirect-dashboard">
+  <template>
+    <div class="message">
+      <h3>The profile plugin has moved.</h3>
+      <p>
+        Please install the new version of the profile plugin from PyPI by
+        running the following command from the machine running TensorBoard:
+      </p>
+      <textarea id="commandTextarea" readonly on-blur="_removeCopiedMessage">
+[[_installCommand]]</textarea
+      >
+      <div id="copyContainer">
+        <span id="copiedMessage"></span>
+        <paper-button raised on-tap="_copyInstallCommand"
+          >Copy to clipboard</paper-button
+        >
+      </div>
+    </div>
+
+    <style include="dashboard-style"></style>
+    <style>
+      .message {
+        margin: 80px auto 0 auto;
+        max-width: 540px;
+      }
+      #commandTextarea {
+        margin-top: 1ex;
+        padding: 1ex 0;
+        resize: vertical;
+        width: 100%;
+      }
+      #copyContainer {
+        display: flex;
+      }
+      #copiedMessage {
+        align-self: center;
+        flex-grow: 1;
+        font-style: italic;
+        padding-right: 1em;
+        text-align: right;
+      }
+    </style>
+  </template>
+
+  <script>
+    (function() {
+      Polymer({
+        is: 'tf-profile-redirect-dashboard',
+        properties: {
+          _installCommand: {
+            type: String,
+            readonly: true,
+            value: [
+              'pip install --extra-index-url https://test.pypi.org/simple/ \\',
+              '    tensorboard_plugin_profile',
+            ].join('\n'),
+          },
+        },
+        _copyInstallCommand() {
+          const tryCopy = (async () => {
+            const textarea = this.$.commandTextarea;
+            textarea.select();
+            try {
+              await navigator.clipboard.writeText(textarea.textContent);
```

Comment:
naive question: wouldn't it be better to just use `this. _installCommand` instead of reading the DOM (even if it is readonly)? Is it because we already select the `textNode` in the textarea for the `execCommand` case? If answer is, 'DOM should be SOT', then would it be nicer to inline content of `_installCommand` onto the template instead of having the property?